requirements.txt

```diff
@@ -30,4 +30,9 @@ wrapanapi==3.6.0
 # Get airgun, nailgun and upgrade from master
 git+https://github.com/SatelliteQE/airgun.git@master#egg=airgun
 git+https://github.com/SatelliteQE/nailgun.git@master#egg=nailgun
+# Broker currently is unable to push to PyPi due to [1] and [2]
+# In the meantime, we install directly from the repo
+# [1] - https://github.com/ParallelSSH/ssh2-python/issues/193
+# [2] - https://github.com/pypi/warehouse/issues/7136
```

Comment:
I think both of these issues have been resolved now, so can we keep this the way it is?

Reply:
@Gauravtalreja1 https://github.com/ParallelSSH/ssh2-python/issues/193 is still open.